IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,288 @@
+Since the research and development is dependent on the payload chosen and the project is still in the brainstorming phase,
+the timeline has been the major guide thus far. It is as follows:
+\begin{itemize}
+  \item September 1st: Begin narrowing down payload ideas using a decision matrix based on the SDL Payload Challenge rubric
+  \item September 8th: Settle on the specific payload and make a plan of action for research and design
+  \item September 30th: Present a preliminary design with estimates for size, cost, and weight
+  \item November 1st: Mechanical design and delivery date to LI. No changes to the body can be made after this point
+  \item Semester's End: Final hardware design completed and materials ordered
+\end{itemize}
+
+This timeline provides roughly 1 month until a preliminary design is created. After this, there will be another month to refine
+the design before the final size and interface details must be provided to LI. After this point, there will be more internal
+hardware design and development for the remainder of the semester with a goal of purchasing all materials by the semester's end.
+If this timeline can be adhered to, then the entire spring semester and summer months leading up to the competition can be
+used for manufacturing, programming, testing, and characterization.
+
+\subsection{Deliverables}
+\label{subsec:Deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+The first deliverable will be the preliminary design and bill of materials with a corresponding estimate of size, weight, and
+cost. If it is found that additional funds need to be raised, then the preliminary design can be used in fundraising requests
+so that businesses and investors know that sufficient research and development has already been put into the project. After the
+preliminary designs, the next deliverable will be the final external dimensions and interfaces so that LI can proceed with their
+construction of the rocket. Finally, the actual payload will need to be delivered by the competition date although it will be provided
+much earlier than that due to the necessity of integration and testing.
+
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+To participate in this endeavor, there are no prerequisite skills. However, for the project to succeed, there will need to be
+individuals who are proficient in computer-aided design and analysis for the mechanical structure of the payload. There will
+also need to be individuals proficient in electronic design, specifically with respect to printed circuit boards. Additionally,
```

Comment:
Is it possible to get this done without these skills? Is it mission critical to avoid using COTS arduinos and breakout boards?